### DISCUSSION TITLE
floret-vecs as drop-in replacement in _sm models

### DISCUSSION THREAD

**Discussion by concepttt-ma** [explosion/spaCy#10545] [Help: Coding & Implementations] [answered]
Just a brief question: is it possible to use one of the _sm pretrained models (for taggin, parsing, ner,..) and have the model yield floret vectors (which I have already trained and created a spacy model for)? 
I mean, the model should then access the floret vecs when the .vector (property) is called on the tokens.

Of course, I could use an extension to get/set the floret-vector if needed (with an "additional" spacy model with floret vecs only); or are the user_hooks a more apt tool for this? Anyway, I was just wondering if there is an easy way to create such a hybrid model.

Any help is appreciated
have a good day

**Comment by adrianeboyd** [comment, marked as answer]
Sure, as long as the model doesn't currently have vectors you should be able to do:

```python
floret_nlp = spacy.load("floret_model")
nlp = spacy.load("en_core_web_sm")
nlp.vocab.vectors = floret_nlp.vocab.vectors
```

**Reply by concepttt-ma** [reply]
Hi Adriane
Thanks for the lightening fast answer! 
ok, cool that works - it would still imply that I start both models alongside, right? (guess, that this replacement has to be done at runtime and is not configurable on the level of the model (let's say a customized version of the _sm model)).

**Reply by adrianeboyd** [reply]
You can save any pipeline with `to_disk` and load from a directory or package with `spacy package` as you wish.

**Reply by concepttt-ma** [reply]
Perfect!
